pipeline/scripts/5/1/tier-1/test-cephadm-apply-service-spec.sh

```diff
@@ -5,7 +5,7 @@ random_string=$(cat /dev/urandom | tr -cd 'a-z0-9' | head -c 5)
 instance_name="ci-${random_string}"
 platform="rhel-8"
 rhbuild="5.1"
-test_suite="suites/pacific/cephadm/tier1_service_apply_spec.yaml"
+test_suite="suites/pacific/cephadm/tier1_5_1_service_apply_spec.yaml"
```

Comment:
Let's have the file name as `tier_1_service_apply_spec_5.1.yaml`

Reply:
As per our discussion, separated out build-version values against test suite name.